invenio_records_resources/services/records/service.py

```diff
@@ -651,13 +651,17 @@ def _create_record(record_dict):
                 _create_record(record_dict)
             except Exception as exc:
                 records_processed.append(("create", record_dict, None, exc))
-
-        # We only commit records that have no errors
-        records = [
-            record
-            for _, record, errors, exc in records_processed
-            if errors == [] and exc is None
-        ]
-        uow.register(RecordBulkCommitOp(records, self.indexer))
+        valid_records = []
+
+        for i, (action, record, errors, exc_) in enumerate(records_processed):
+            if errors == [] and exc_ is None:
+                try:
+                    record.commit()
```

Comment:
can you please add a comment that explains that we cannot use the `uow` because of the error tracking?

Reply:
Yes, added, thanks!